.github/workflows/python_analysis.yml

```diff
@@ -0,0 +1,49 @@
+name: Python analysis
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - develop
+      - main
+      - release/**
+      - feature/**
+      - hotfix/**
+  push:
+    branches:
+      - develop
+      - main
+      - release/**
+      - feature/**
+      - hotfix/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  call-workflow-static-analysis:
+    name: Static analysis
+    uses: MiraGeoscience/CI-tools/.github/workflows/reusable-python-static_analysis.yml@main
+    with:
+      package_manager: 'poetry'
+      app_name: 'omf'
+      python_vers: '3.9'
```

Comment:
**issue:** restrict execution of pylint to `omf/fileio/geoh5* tests`
this repo is a fork, and we do not want to touch the original code

See in olde static-analysis.yml workflow: https://github.com/MiraGeoscience/omf/pull/66/files#diff-e9db86b6fd5f791eb849075b600b921e5e95b978e41f9d290c01027440076ec3L75